docs/install.md

```diff
@@ -60,7 +60,16 @@ To install Compose, do the following:
 
 ### Install using pip
 
-    $ sudo pip install -U docker-compose
+Compose can be installed from [pypi](https://pypi.python.org/pypi/docker-compose)
+using `pip`.  If you install using `pip` it is highly recommended that you use a
+[virtualenv](https://virtualenv.pypa.io/en/latest/) because many operating systems
+have python system packages that conflict with docker-compose dependencies.
```

Comment:
I agree with this comment and us recommending using pip. 

I'm not convinced we should give the steps, below, on how to do it, as there is so much room there for people's personal setup/preferences that I don't think we should be prescribing that.